transcoder/main.py

```diff
@@ -37,10 +39,13 @@
 from transcoder.source import all_source_identifiers
 
 script_dir = os.path.dirname(__file__)
+message_parser: MessageParser = None
 
 
 def main():
     """main entry point for Datacast Transcoder"""
+
+    global message_parser
```

Comment:
The linter doesn't like using global. It appears that signal can be wired up to instances: https://stackoverflow.com/questions/34786524/signal-handler-inside-a-class